aiohttp/connector.py

```diff
@@ -719,6 +720,8 @@ class TCPConnector(BaseConnector):
     ttl_dns_cache - Max seconds having cached a DNS entry, None forever.
     family - socket address family
     local_addr - local tuple of (host, port) to bind socket to
+    network_interface - String of the network interface to bind socket to.
```

Comment:
If you want it to appear in the documentation, you'll need to update docs/reference as well.

It would also be a good idea to give an example. By network interface, do you mean e.g. "eth0"?

Reply:
In which case, it may be better as:
```suggestion
    network_interface - Name of network interface to bind socket to (e.g. "eth0").
```

Reply:
Agreed.

Reply:
I have added examples, found the docs and changed that as well.